### ISSUE TITLE
Assignment 18

### ISSUE THREAD

**Issue by FSGabrsek** [FSGabrsek/AE-I_Intro-Data-Science#24]
### Regression on personal dataset

- [x] Split the dataset into a training and test set
- [x] Fit a DecisionTreeClassifier with a valid training data
- [x] Document the accuracy of the tree for the chosen set size and depth
- [x] Plot the tree
